pulp_ansible/app/viewsets.py

```diff
@@ -1189,27 +1187,20 @@ class AnsibleDistributionViewSet(DistributionViewSet, RolesMixin):
     }
 
 
-class TagViewSet(NamedModelViewSet, mixins.ListModelMixin):
+class TagViewSet(viewsets.GenericViewSet, mixins.ListModelMixin):
```

Comment:
Any particular reason for changing this from a NamedModelViewSet?

Reply:
Yes, there's no model for it anymore.
And Pulp's viewset chasing magic goes boom if it finds two viesets for `CollectionVersion`.